cmd/homi/setup/cmd.go

```diff
@@ -313,6 +315,12 @@ func genCypressCommonGenesis(nodeAddrs, testAddrs []common.Address) *blockchain.
 	assignExtraData := genesis.Validators(nodeAddrs...)
 	assignExtraData(genesisJson)
 
+	// To test storage trie with Cypress Chaindata
+	genesisJson.Alloc[genesis.StorageTrieTestAddr] = blockchain.GenesisAccount{
```

Comment:
Is this okay? Isn't it produce a different `genesis.json` from the real one?

Reply:
`genCypressGenesis` is the one we use to make a real Cypress genesis.json? If so, this can be a problem as you noted. However, this PR won't be merged in the near future and will be cherry-picked if needed. Please check our Jira issue 😄

Reply:
Nope, the hard-coded genesis file exists to launch cypress/baobab node. I meant it *can* make the same json file using `homi`. As far as I remembered, the cypress genesis file can be (or planned to be) made with different options such as `governance` options, and so on. I checked the Jira issue and understood what you said. :)